pages/configuration/https.rst

```diff
@@ -89,7 +102,7 @@ The resulting ``graylog-certificate.pem`` and ``graylog-key.pem`` can be used in
 Converting an existing Java Keystore to private key/certificate pair
 ====================================================================
 
-This section describes how to export a private key and certificate from an existing Java KeyStore in JKS format.
+This section describes how to export a private key and certificate from an existing Java KeyStore in JKS format. This is needed if you want to get the certificates exported from the Java KeyStore. 
```

Comment:
Try instead: "This is needed if you want to export the certificates from the Java KeyStore."